### PR TITLE
Historical data insertion

### PR DESCRIPTION
**Pull request by aaronreynoza**
This service inserts historical transactions in a Redis store based on ILP.

### CODE REVIEW

File: historical-data-store/src/helper.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const log = (data: any) => {
+  // eslint-disable-next-line no-console
+  console.log(data);
```
Comment (johanfol) [inline historical-data-store/src/helper.ts:3]: This must log to Kafka topic ruleEngineLogs - not console - this is very slow.

File: historical-data-store/src/redis-client.ts [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+    }
+    resolve(data);
+  });
+});
+
+const logAllkeys = async (client: RedisClient) => client.keys('*', (err, keys) => {
+  if (err) {
+    log(err);
+  } else {
+    log(`stored keys are: ${JSON.stringify(keys)}`);
```
Comment (johanfol) [inline historical-data-store/src/redis-client.ts:82]: Yeah... no. haha, rather log like the count, or something, not the whole thing?

**Review by johanfol — CHANGES_REQUESTED** [review]

File: historical-data-store/docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+version: "2"
+services:
+  redis:
+    image: redis
```
[the remark this answers is not in the document]
Comment (aaronreynoza) [reply historical-data-store/docker-compose.yaml:4]: I actually set this up to run redis locally in the meantime. Once we have the k8s cluster running I'm planning to remove all the docker-compose files. we should probably discuss this.

File: historical-data-store/package.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "blocklist-processing-engine",
```
[the remark this answers is not in the document]
Comment (aaronreynoza) [reply historical-data-store/package.json:2]: touche

**Pull request merged**